README.md

```diff
@@ -2,99 +2,41 @@
 [![codecov](https://codecov.io/gh/pendulum-project/ntpd-rs/branch/main/graph/badge.svg?token=WES1JIYUJH)](https://codecov.io/gh/pendulum-project/ntpd-rs)
 [![Crates.io](https://img.shields.io/crates/v/ntpd.svg)](https://crates.io/crates/ntpd)
 
-
 # ntpd-rs
 
-ntpd-rs is an NTP implementation written in Rust, with a focus on security and
-stability. It includes client and server functionality and supports NTS.
+nptd-rs is a tool for synchronizing your computer's clock. Implementing the NTP and NTS protocols, it is written in Rust, with a focus on security and stability. It includes both client and server support.
 
 If a feature you need is missing please let us know by opening an issue.
 
-## Installation
-
-The recommended way of installing ntpd-rs is with the pre-built packages from
-the [releases page]. The installers automatically handle setting up users,
-permissions and configuration. Alternatively, you can use `cargo install ntpd`
-or build from source by cloning the repository yourself.
-
-### Build from source
-
-Currently, ntpd-rs only supports Linux-based operating systems. Our current
-testing only targets Linux kernels after version 5.0.0, older kernels may work
-but this is not guaranteed.
-
-ntpd-rs is written in rust. We strongly recommend using [rustup] to install a
-rust toolchain, because the version provided by system package managers tends to
-be out of date. Be sure to use a recent version of the rust compiler. To build
-ntpd-rs run
-
-```sh
-cargo build --release
+## Use
+
+After installing the packages from the [releases page], ntpd-rs will by default synchronize your computers clock to servers from the [NTP pool]. When installed, you can check the synchronization status through
+```console
+$ ntp-ctl status
+Synchronization status:
+Dispersion: 0.000299s, Delay: 0.007637s
+Desired poll interval: 16s
+Stratum: 4
+
+Sources:
+ntpd-rs.pool.ntp.org:123/77.171.247.180:123 (1): +0.000024±0.000137(±0.016886)s
+    pollinterval: 16s, missing polls: 0
+    root dispersion: 0.005905s, root delay:0.016190s
+ntpd-rs.pool.ntp.org:123/45.137.101.154:123 (2): +0.000022±0.000081(±0.007414)s
+    pollinterval: 16s, missing polls: 0
+    root dispersion: 0.004517s, root delay:0.005051s
+ntpd-rs.pool.ntp.org:123/178.215.228.24:123 (3): +0.000117±0.000091(±0.009162)s
+    pollinterval: 16s, missing polls: 0
+    root dispersion: 0.000549s, root delay:0.004318s
+ntpd-rs.pool.ntp.org:123/162.159.200.123:123 (4): +0.000111±0.000076(±0.004066)s
+    pollinterval: 16s, missing polls: 0
+    root dispersion: 0.000351s, root delay:0.003571s
```

Comment:
it it's "root dispersion"  then should it not also be "poll interval"? (i know that is not in this PR but I'm noticing it now)

Reply:
made this change in https://github.com/pendulum-project/ntpd-rs/pull/1101